### PR TITLE
UI changes according to available designs in zeplin

### PR DESCRIPTION
**Pull request by dewanshDT**
Changes:
designimprovements

image

These changes are made in accordance with the [available designs](https://scene.zeplin.io/project/55f746c54a02e1e50e0632c3).
I implement this because these subtle changes look much better than the UI currently in use.

looking forward to your feedback on this @raclim @lindapaiste

I have verified that this pull request:

* [x] has no linting errors (`npm run lint`)
* [x] has no test errors (`npm run test`)
* [x] is from a uniquely-named feature branch and is up to date with the  `develop` branch.
* [x] is descriptively named and links to an issue number, i.e. `Fixes #123`

### CODE REVIEW

File: server/views/index.js [hunk 1]
```diff
@@ -23,25 +28,51 @@ export function renderIndex() {
...
+          process.env.UPLOAD_LIMIT ? `${process.env.UPLOAD_LIMIT}` : undefined
+        };
+        window.process.env.MOBILE_ENABLED = ${
+          process.env.MOBILE_ENABLED
+            ? `${process.env.MOBILE_ENABLED}`
+            : undefined
+        };
+        window.process.env.TRANSLATIONS_ENABLED = ${
+          process.env.TRANSLATIONS_ENABLED === 'true' ? true : false
+        };
```
Comment (3ru) [inline server/views/index.js:63]: These unnecessary differences should be removed.

Or was the formatting not applied before?

**Review by 3ru — CHANGES_REQUESTED** [review]
It would be nice to have pictures of the UI changes before and after.

File: server/views/index.js (hunk 1 above)
Comment (dewanshDT) [reply server/views/index.js:63]: the formatting was not applied before
and the screenshots are captured after the changes.

**Comment by dewanshDT** [comment]
this is before and after the changes.
### Before:
image

### After:
smalldesignimprovements

changes made with reference to [these design files](https://scene.zeplin.io/project/55f746c54a02e1e50e0632c3)

hey @3ru check this out, any suggestions are welcome 😃

**Comment by raclim** [comment]
Thanks so much for taking the time to work on this, I really appreciate it! 

Since the zeplin designs are a little outdated, we were considering doing another redesign in the future. With this in mind, I don't think we're ready to make these UI changes yet, so I'm going to close this for now. We'll probably post an issue regarding any design updates, so please feel free to reference and work on those when it happens!

**Pull request closed without merging**